templates/zerver/help/see-if-user-here-away.md

```diff
@@ -0,0 +1,35 @@
+### User List
+
+You can see if a user is active on Zulip by looking at the user list located on the far right side of your screen.
+
+![User List](/static/images/help/user-list.png)
+
+The **Search people** field at the top of the user list allows you to search for a specific user. As you type the user's name, the user list will shrink to show only people with that name.
```

Comment:
The user list starts shrinking as soon as you start typing, meaning it shows only users whose names match the search query - could you address that? You don't have to type the whole name to shrink the list.